### ISSUE TITLE
Is testing with the S3 protocol now supported?

### ISSUE THREAD

**Issue by humengyu2012** [mlcommons/storage#78]

**Comment by FileSystemGuy** [comment]
No, not as part of CLOSED submission, but it is possible using an OPEN submission.  The WG is currently debating the features we will include in v2.0 of the benchmark and S3 is on the list.  It is more of a prioritization question than a "do we need it" question, so if you think it should be on the committed list, please attend the WG meetings and/or send email to the alias to make your case that it should be committed for v2.0.

**Comment by FileSystemGuy** [comment]
You are welcome to join the [Working Group](https://mlcommons.org/working-groups/benchmarks/storage/), that will get you added to the email list that we use: storage@mlcommons.org.  You can email the list directly and get responses, but you won't see all of the other discussion happening within the group.

**Comment by FileSystemGuy** [comment]
This is moot (no longer relevant) so is being closed.
